elasticsearch/init.sh

```diff
@@ -18,19 +18,27 @@
 source "logging"
 
 wait_for_port_open
-failed=0
-
-touch ${HOME}/init_running
-pushd "${ES_HOME}/init"
-  files=$(ls --hide common | sort)
-  for init_file in ${files}; do
-    if [ -f "${init_file}" ] ; then
-      ./"${init_file}" || { failed=1 ; echo "failed init: ${init_file}" >> ${HOME}/init_failures ; }
-    fi
-  done
-popd
-
-rm -f ${HOME}/init_running
-if [ $failed -eq 0 ]; then
-  touch ${HOME}/init_complete
+
+# find out who the master of the cluster is
+master_node=$(get_master_node)
```

Comment:
Do we have any concerns about the outcome here?  Is there possibly a delay between the running of the script and the election of the master?

Reply:
not sure... since this is after `wait_for_port_open` i would guess its a small window.. but I could always add logic in that function to ensure we have a value first

Reply:
i just saw this on my cluster... 
```
jq: error: CShWUf/0 is not defined at <top-level>, line 1:
.nodes.apRFr1-CShWUf-_ZIQOWbw.name              
jq: error: _ZIQOWbw/0 is not defined at <top-level>, line 1:
.nodes.apRFr1-CShWUf-_ZIQOWbw.name                     
jq: 2 compile errors
(23) Failed writing body
```
I'll try to add some hardening